### DISCUSSION TITLE
来个大佬帮忙分析一下性能占用

### DISCUSSION THREAD

**Discussion by E0x72-21** [Meloong-Git/PCL#5027] [提问] [answered]
https://spark.lucko.me/25Qki0xhBV
https://spark.lucko.me/pWCYEQJ5Yj

求问是什么模组或者设置导致后者cpu的占用更高

以下是两个性能报告其中不一样的模组
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Eric/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Eric/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

25Qki0xhBV |   | pWCYEQJ5Yj
-- | -- | --
Bad-Horse-Fix-2.0.0-fabric.jar |   | badhorsefix-2.1.0-fabric.jar
BadOptimizations-2.1.4-1.21.jar |   | BadOptimizations-2.2.0-1.21.1.jar
caxton-0.6.0-alpha.2+1.21.1-FABRIC.jar |   | caxton-0.6.0-alpha.3+1.21.1-FABRIC.jar
cicada-lib-0.8.3+1.21-and-above.jar |   | cicada-lib-0.9.0+1.21-1.21.1.jar
clientcommands-2.9.1.jar |   | clientcommands-2.9.3.jar
do_a_barrel_roll-fabric-3.6.2+1.21.jar |   | do_a_barrel_roll-fabric-3.7.2+1.21.jar
emi-1.1.13+1.21.1+fabric.jar |   | emi-1.1.16+1.21.1+fabric.jar
fabric-api-0.104.0+1.21.1.jar |   | fabric-api-0.107.0+1.21.1.jar
fabric-language-kotlin-1.12.1+kotlin.2.0.20.jar |   | fabric-language-kotlin-1.12.3+kotlin.2.0.21.jar
fast-ip-ping-v1.0.4-mc1.21.1-fabric.jar |   | fast-ip-ping-v1.0.5-mc1.21.1-fabric.jar
ImmediatelyFast-Fabric-1.2.21+1.21.1.jar |   | ImmediatelyFast-Fabric-1.3.0+1.21.1.jar
libIPN-fabric-1.21-6.0.2.jar |   | libIPN-fabric-1.21-6.0.3.jar
Searchables-fabric-1.21.1-1.0.1.jar |   | Searchables-fabric-1.21.1-1.0.2.jar
smoothskies-2.0.1.jar |   | smoothskies-2.0.4+1.20.4-fabric.jar
tweakermore-v3.20.1-mc1.21.jar |   | tweakermore-v3.21.0-mc1.21.1.jar
ViaFabricPlus-3.4.5.jar |   | ViaFabricPlus-3.4.9.jar
vmp-fabric-mc1.21.1-0.2.0+beta.7.167-all.jar |   | vmp-fabric-mc1.21.1-0.2.0+beta.7.168-all.jar
YetAnotherConfigLib-3.5.0+1.21-fabric.jar |   | YetAnotherConfigLib-3.6.1+1.21-fabric.jar
[Xaero的小地图]   Xaeros_Minimap_24.4.0_Fabric_1.21.jar |   | [Xaero的小地图]   Xaeros_Minimap_24.5.0_Fabric_1.21.jar
[一键背包整理Next]   InventoryProfilesNext-fabric-1.21-2.0.4.jar |   | [一键背包整理Next]   InventoryProfilesNext-fabric-1.21-2.0.7.jar
[动态FPS]   dynamic-fps-3.6.3+minecraft-1.21.0-fabric.jar |   | [动态FPS]   dynamic-fps-3.7.4+minecraft-1.21.0-fabric.jar
[模组菜单]   modmenu-11.0.2.jar |   | [模组菜单]   modmenu-11.0.3.jar
[简单的语音聊天]   voicechat-fabric-1.21.1-2.5.22.jar |   | [简单的语音聊天]   voicechat-fabric-1.21.1-2.5.25.jar
[聊天记录保存]   permanent-chat-logs-0.2.3-build20240714.jar |   | [聊天记录保存]   permanent-chat-logs-0.3.1-build20241013.jar
[铁氧体磁芯]   ferritecore-7.0.0-fabric.jar |   | [铁氧体磁芯]   ferritecore-7.0.2-hotfix-fabric.jar
[锂]   lithium-fabric-mc1.21.1-0.13.0.jar |   | [锂]   lithium-fabric-0.14.0-snapshot+mc1.21.1-build.88.jar
[键位冲突显示]   Controlling-fabric-1.21.1-19.0.2.jar |   | [键位冲突显示]   Controlling-fabric-1.21.1-19.0.3.jar
  |   | architectury-13.0.6-fabric.jar.disabled
  |   | GpuTape-1.18x-1.21x-1.0.3.jar.disabled
  |   | idle_boost-fabric-1.2.0+1.21.1.jar.disabled
  |   | threatengl-fabric-1.3.3-release.1.jar

</body>

</html>

**Comment by llsgllsg** [comment]
这CPU占用不是很低吗
![image](https://github.com/user-attachments/assets/1a2a4d7e-b48c-4aa0-81b6-010a144a671f)

**Reply by E0x72-21** [reply]
那为什么我U的温度从平均7x℃蹦到8x℃了？都暴增10℃了

**Reply by E0x72-21** [reply]
> 那为什么我U的温度从平均7x℃蹦到8x℃了？都暴增10℃了

在菜单界面都像在烤机一样吃满cpu了
![image](https://github.com/user-attachments/assets/c3461ca8-63ec-45e3-9bdb-83075ef8541b)

**Comment by E0x72-21** [comment, marked as answer]
问题解决了，[聊天记录保存] permanent-chat-logs-0.3.1-build20241013.jar的锅

https://spark.lucko.me/CRVrM4C3fa

**Comment by llsgllsg** [comment]
暴增10℃不是再正常不过的事情吗
